loadtestconfig.rst

```diff
@@ -0,0 +1,571 @@
+    
+    -example    ``"LowVolumeTeamSelectionWeight":1``
+    
+
+HighVolumeChannelSelectionWeight
+-----------------------------------------------------------------------------
+
+    ``HighVolumeChannelSelectionWeight`` is the probability of selection of a *high volume*  channel.
+    
+    -example    ``"HighVolumeChannelSelectionWeight":1``
+    
+    
+
+MidVolumeChannelSelectionWeight
+-----------------------------------------------------------------------------
+
+    ``MidVolumeChannelSelectionWeight`` is the probability of selection of a *medium volume*  channel.
+    
+    -example    ``"MidVolumeChannelSelectionWeight":3``
+    
+    
+
+LowVolumeChannelSelectionWeight
+-----------------------------------------------------------------------------
+
+    `LowVolumeChannelSelectionWeight`` is the probability of selection of a *low volume*  channel.
+    
+    -example    ``"LowVolumeChannelSelectionWeight":1``
+
+   
+
+
+User Entities Configuration
+==============================
+
+
+An *entity* in the ``loadtestconfig.json`` configuration file is a boundary or limitation applied to the test.
+
+The variables are as follow below.
+
+
+TestLengthMinutes
+-------------------------------------
+
+The ``TestLengthMinutes`` variable determines the length of the test in minutes. 
+
+    -example    ``"TestLengthMinutes":20``
+    
+
+NumActiveEntities
+---------------------------------
+
+    The ``NumActiveEntities`` variable determines how many entities are run. 
+    This should be set to your number of expected active users.
+
+    -example    ``"NumActiveEntities":500"``
+
+ActionRateMilliseconds
+---------------------------------------------
+
+    The ``ActionRateMilliseconds`` variable determines how often each entity should take an action. 
+
+    -example    ``"ActionRateMilliseconds" :60000``
+    
+.. tip::For an entity that only posts this would be the time between posts.
+    
+
+ActionRateMaxVarianceMilliseconds
+-------------------------------------------------------
+
+    ``ActionRateMaxVarianceMilliseconds`` is the maximum variance in action rate for each wait period. 
+
+
+    -example    ``"ActionRateMaxVarianceMilliseconds":1500``
+    
+    
+
+EnableRequestTiming
+-----------------------------------------
+
+     ``EnableRequestTiming``  enables timing for requests. 
+          
+
+    -options    true/false
+    -default    true
+    -example    ``"EnableRequestTiming":true``
+    
+
+UploadImageChance
+--------------------------------------
+
+    ``UploadImageChance`` is the chance an image will be uploaded.
+    
+    
+    -example    ``"UploadImageChance":0.01``
+     
+     
+     
+.. note:: Chance here is used in the same way as *weight* in the ``loadtestconfig.json`` file. It means a probability deifned by the numer allocated to this variable. The higher the number the higher the chance. In this instance the *chance* is the uploading of an image. 
```

Comment:
No it's not the same as weight. It is the literal probability of an image being uploaded with each post.